src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlClientEventSource.cs

```diff
@@ -285,6 +285,11 @@ internal void TraceEvent<T0>(string message, T0 args0)
                 Trace(string.Format(message, args0?.ToString() ?? "Null"));
             }
         }
+        [NonEvent]
+        internal void TraceEventExcluded<T0, T1>(string message, T0 args0, T1 args1)
```

Comment:
We have another SqlClientEventSource.cs here:
src\Microsoft.Data.SqlClient\netcore\src\Microsoft\Data\SqlClient\SqlClientEventSource.cs

Reply:
I will remove that.